cmd/release/utils/values/searchprs.go

```diff
@@ -12,34 +12,53 @@ const (
 	githubClient := github.NewClient(nil)
 
 	ctx := context.Background()
-	opts := &github.SearchOptions{}
+	opts := &github.SearchOptions{Sort: "updated"}
 	//Get the date of the last document release for the release version
-	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:" + releaseVersion, opts)
+	prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged label:release label:documentation repo:aws/eks-distro label:"+releaseVersion, opts)
 	if err != nil {
 		return "", fmt.Errorf("getting PRs from %v: %w", githubClient, err)
 	}
 
-	//Select the most recent pr from the above query and format the date expected for the go-github client
-	lastDocRelease := prs.Issues[0].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+	lastDocRelease := "2006-01-02T15:04:05+00:00"
+	prevDocRelease := "2006-01-02T15:04:05+00:00"
+	if len(prs.Issues) > 0 {
+		//Select the most recent pr from the above query and format the date expected for the go-github client
+		lastDocRelease = prs.Issues[0].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+	} else {
+		//With no document releases we need to be a little bit clever to generate unannounced changelogs.
+		//This finds the
+		opts = &github.SearchOptions{Sort: "updated", Order: "asc"}
+		prs, _, err := githubClient.Search.Issues(ctx, "is:pr is:merged 'Bumped Production release number' in:title label:release label:"+releaseVersion, opts)
+		if err != nil {
+			return "", fmt.Errorf("get PRs from %v: %w", githubClient, err)
+		}
+		if overrideNumber == 1 {
+			lastDocRelease = prs.Issues[overrideNumber-1].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+		} else {
+			lastDocRelease = prs.Issues[overrideNumber-1].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+			prevDocRelease = prs.Issues[overrideNumber-2].ClosedAt.Format("2006-01-02T15:04:05+00:00")
+		}
+
+	}
 
 	patchPRs, _, err := githubClient.Search.Issues(ctx,
-		fmt.Sprintf("%v merged:>%v label:patch label:%v", baseQuery, lastDocRelease, releaseVersion), opts)
+		fmt.Sprintf("%v merged:%v..%v label:patch label:%v", baseQuery, prevDocRelease, lastDocRelease, releaseVersion), opts)
 	if err != nil {
 		return "", fmt.Errorf("getting patch prs: %w", err)
 	}
 
 	baseImgPRs, _, err := githubClient.Search.Issues(ctx,
-		fmt.Sprintf("%v merged:>%v label:base-img-pkg-update label:%v", baseQuery, lastDocRelease, releaseVersion), opts)
+		fmt.Sprintf("%v merged:%v..%v label:base-img-pkg-update label:%v", baseQuery, prevDocRelease, lastDocRelease, releaseVersion), opts)
 	if err != nil {
 		return "", fmt.Errorf("getting base image prs: %w", err)
 	}
 
-	versPRs, _, err := githubClient.Search.Issues(ctx, fmt.Sprintf("%v merged:>%v label:project label:%v",
-		baseQuery, lastDocRelease, releaseVersion), opts)
+	versPRs, _, err := githubClient.Search.Issues(ctx, fmt.Sprintf("%v merged:%v..%v label:project label:%v",
+		baseQuery, prevDocRelease, lastDocRelease, releaseVersion), opts)
```

Comment:
Nit: can we not use `Doc` the name `prevDocRelease`? There is a 100% chance I'm going to forget how this works and erroneously think it's going off the docs PR